app/views/gds_main_template.scala.html

```diff
@@ -27,7 +27,7 @@
         phaseBanner: components.phaseBanner
 )
 
-@(  title: Title,
+@(  title: String,
```

Comment:
This change require all pages using this component to be updated. Please go through all pages that are using it and make sure the title is displayed correctly.
As a side note: if tests have not caught thi, it might be that they're using `stubMessages` from Play! helpers. This is not what we really want, because this stub has no mapping for any key, so it always returns the key itself. You may end up displaying keys only and the tests won't tell you that.
What you want to use is `MessagesStub` trait.

Reply:
Ignore Maciej (just this once) - "This change require all pages using this component to be updated. Please go through all pages that are using it and make sure the title is displayed correctl" .